packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -377,7 +406,7 @@ function returnModalCopy(
 								'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. Since your other product(s) depend on your plan to be purchased, they will also be removed from the cart and we will take you back to your site.' )` *ES Score: 9*